### PR TITLE
Fr15 monthview

### PR DESCRIPTION
**Pull request by Shandorin**
Various fixes/changes.

### CODE REVIEW

File: front/src/utils/Utils.js [hunk 1]
```diff
@@ -129,7 +129,32 @@ export function viewChanger() {
     return table;
   } catch (err) {
     console.error(err);
-    return false;
+    const date = new Date(Date.now());
+    console.log(date);
+    const time = moment(`${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`, 'YYYY-MM-DD');
+    console.log(time);
```
Comment (TuoLuo994) [inline front/src/utils/Utils.js:135]: remove consolelogs

Comment (Shandorin) [reply front/src/utils/Utils.js]: Done

**Comment by TuoLuo994** [comment]
Could you add monthview tests? Check out front-tests branch for examples.

**Comment by Shandorin** [comment]
I'l add tests on weekend

File: front/src/utils/Utils.js [hunk 2]
```diff
@@ -129,7 +129,30 @@ export function viewChanger() {
     return table;
   } catch (err) {
     console.error(err);
-    return false;
+    const date = new Date(Date.now());
```
Comment (Liljenmaa) [inline front/src/utils/Utils.js:132]: I'm not sure but Date.now() should return a new Date by default

**Review by Liljenmaa — APPROVED** [review]
Codewise looks good. After tests it may be merged normally.

**Pull request merged**